doc/widgets/explain-pred.md

```diff
@@ -0,0 +1,35 @@
+Explain Prediction
+==================
+
+Explains which features contribute the most to the prediction for a single instance based on the model and how they contribute.
+
+**Inputs**
+
+- Model: a model whose predictions are explained by the widget
+- Background data: data needed to compute explanations
+- Data: Single data instance whose prediction is explained by the widget
+
+**Outputs**
+
+- Scores: The SHAP value of each features value. Features that contribute more to prediction have higher score deviation from the 0.
+
+**Explain Prediction** widget explains classification or regression model's prediction for the provided data instance. The widget shows what features affect the prediction of selected class the most and how they contribute (towards or against the prediction). The explanation is computed with removing features, replacing them with different options from the background data, and observing the change in the prediction.
+![](images/Explain-Prediction.png)
+
+1. Select the target class -- the plot will show explanations for this class.
+2. Zoom in/out the plot.
+3. Observe the prediction probability for a class and base value -- an average probability in the dataset.
+4. Plot which shows features that affect the prediction the most (features with longer tape length) and how they affect it. Red features increase the probability for a selected class while blue features decrease the probability. On the right from the tape, you can see the feature name and its value\* for the selected instance. The length of the tape segment (and number on the tape) represent the SHAP value for feature contribution -- it is how much the feature affects the probability for the selected class. Edino mogoče bi še dodala, kaj sta ti dve sivi številki na levi. 0.96 in 0.45 na primeru iz dokumentacije. Numbers in the gray boxes indicate the prediction probability for the selected class is (0.6) and the baseline probability (0.45) (the average probability in the data).  
```

Comment:
> Edino mogoče bi še dodala, kaj sta ti dve sivi številki na levi. 0.96 in 0.45 na primeru iz dokumentacije.

😆

Reply:
Haha. I accidentally pasted wrong text and then it seems removed just part of it. Now it should be ok